src/reducers/__tests__/splitSentences.js

```diff
@@ -0,0 +1,42 @@
+import { ROOT_TOKEN } from '../../constants'
+import { initialState, reducerFlow } from '../../util'
+import { exportContext } from '../../selectors'
+
+import { newThought, splitSentences } from '../../reducers'
+import setCursorFirstMatch from '../../test-helpers/setCursorFirstMatch'
+
+it('split sentences with only one thought', () => {
+  const steps = [
+    newThought('One. Two. Three.'),
+    setCursorFirstMatch(['One. Two. Three.']),
```

Comment:
`newThought` automatically sets the cursor to the newly created thought as long as `preventSetCursor` is not set to `true`, so you can omit `setCursorFirstMatch` here.